pysat/instruments/methods/nasa_cdaweb.py

```diff
@@ -285,6 +291,7 @@ def download(supported_tags, date_array, tag, sat_id,
                                 date.strftime('%d %B %Y'))))
                 sys.stdout.flush()
                 remote_files = list_remote_files(tag=tag, sat_id=sat_id,
+                                                 remote_site=remote_site,
```

Comment:
Do we have a potential issue now that times within a day are included in the formatted names but not in the list_remote_files method?

Reply:
github was hiding code and I initially missed we are in a single file per day situation.

Reply:
That's right, this bit is only for multi-file days